src/charts/CytoViz/CytoViz.js

```diff
@@ -131,19 +131,21 @@ export const CytoViz = (props) => {
 
   const errorBanner = error && <ErrorBanner error={error} labels={labels_.errorBanner} />;
   const loadingPlaceholder = loading && !error && !placeholderMessage && (
-    <div className={classes.loadingContainer}>
+    <div data-cy="cytoviz-loading" className={classes.loadingContainer}>
```

Comment:
Cant we rename this selector to `citoviz-loading-container` to name the selector, not action ? :slightly_smiling_face:

Reply:
Good idea, thank you for the suggestion :)